scripts/vim-patch.sh

```diff
@@ -107,9 +107,12 @@ commit_message() {
   fi
 }
 
-find_git_remote() {
-  git remote -v \
-    | awk '$2 ~ /github.com[:\/]neovim\/neovim/ && $3 == "(fetch)" {print $1; exit}'
+set_git_remote() {
+  git_remote=$(git remote -v \
```

Comment:
this works on `local` variables?!

Reply:
> this works on local variables?!

If the call is in the local scope, yes.

Will change it to echo "origin" then.